File: Greedy/455. Assign Cookies.py
class Solution:
    def findContentChildren(self, g: List[int], s: List[int]) -> int:
        g.sort()
        s.sort()
        i = 0
        for j in s:
            if j >= g[i]:
                i += 1
            if i == len(g):
                return i
        return i

File: Greedy/455. Assign Cookies.cpp
class Solution {
public:
    int findContentChildren(vector<int>& g, vector<int>& s) {
        sort(g.begin(), g.end());
        sort(s.begin(), s.end());
        int i = 0;
        for (int j = 0; j < size(s); j++){
            if (g[i] <= s[j]){
                i++;
            }
            if (i == size(g)){
                return i;
            }
        }
        return i;
    }
};